docs/release-notes/release-notes-0.20.0.md

```diff
@@ -28,6 +28,13 @@
 
 # New Features
 
+* The `RegisterConfirmationsNtfn` call of the `ChainNotifier` now optionally
+  [supports returning a confirmation notification for each confirmation
+  received](https://github.com/lightningnetwork/lnd/pull/9878) for the target
+  transaction/output script. Additionally, the `Updates` channel, which was
+  sent on every incremental confirmation, has been removed from the
```

Comment:
I think this is misleading, it should rather state that the Update Channel was refactored out but its functionality is still available and can be now be selected via an optional paramter.